lib/ueberauth/strategy/strava.ex

```diff
@@ -13,22 +15,26 @@ defmodule Ueberauth.Strategy.Strava do
   Handles initial request for Strava authentication.
   """
   def handle_request!(conn) do
+    scopes = conn.params["scope"] || option(conn, :default_scope)
+    opts = [redirect_uri: callback_url(conn), scope: scopes]
 
-    authorize_url = conn.params
```

Comment:
I went with the approach that the more recent strategies use: https://github.com/ueberauth/ueberauth_github/blob/master/lib/ueberauth/strategy/github.ex#L89